### PR TITLE
desktop/dbgl-legacy: new maintainer; builds from source

### PR DESCRIPTION
**Pull request by Ythogtha**
Same as dbgl, we are rebuilding from source.
But here it also requires an old JDK to run, so maybe we could alter the /usr/bin/dbgl script to actively source a /etc/profile.d/some_jdk7or8.sh ?
Since this is a legacy version, here only for 32bit support.

### CODE REVIEW

**Comment by aclemons** [comment]
You have noted it in the README. That's probably enough. Everything in the repo that needs java to build or run has the problem that you can't really install more than one at a time easily as they are in the repo now.

File: desktop/dbgl-legacy/dbgl-legacy.info [hunk 1]
```diff
@@ -1,10 +1,10 @@
 PRGNAM="dbgl-legacy"
 VERSION="0.83"
-HOMEPAGE="http://members.quicknet.nl/blankendaalr/dbgl/"
-DOWNLOAD="https://slackware.uk/~urchlay/src/dbgl083_generic.tar.gz"
+HOMEPAGE="https://dbgl.org/"
+DOWNLOAD="https://dbgl.org/download/src083.zip"
```
Comment (aclemons) [inline desktop/dbgl-legacy/dbgl-legacy.info:4]: I get an MD5SUM error here?

Comment (Ythogtha) [reply desktop/dbgl-legacy/dbgl-legacy.info:4]: Yes, Don't how how it slipped past me, but here is a fixed PR.

**Comment by aclemons** [comment]
```
# buildpkg i586 desktop/dbgl-legacy
sbolint: dbgl-legacy checks out OK
git repo: /var/lib/repo/SBo/slackbuilds
branch:   github
date:     Sat May 20 00:09:36 GMT 2023
revision: 3e9dbe9a2aa3f856a6b02b099d93e89a545f0b98
title:    desktop/dbgl-legacy: new maintainer; builds from source

================================================================================
desktop/dbgl-legacy                                                     00:09:41
================================================================================

Calculating dependencies ... 
Hints for development/zulu-openjdk8:
  PRAGMA="x86arch=i686"
Dependency tree:
desktop/dbgl-legacy (update for git 3e9dbe9 "new maintainer; builds from source")
  system/dosbox (ok)
  development/zulu-openjdk8 (ok)
  development/apache-ant (ok)

----------------------------------------------------------------------- 00:09:41
Building desktop/dbgl-legacy (update for git 3e9dbe9 "new maintainer; builds from source")
Testing SlackBuild files ... done.
Downloading source files ...
Verifying source files ... done.
Installing dependencies ...
apache-ant-1.10.12-noarch-1_SBo: Java-based build tool ................. [  42M]
  Running profile script: /etc/profile.d/apache-ant.sh
zulu-openjdk8-8.0.362-i686-2_SBo: Open Implementation of JDK ........... [ 209M]
  Running profile script: /etc/profile.d/zulu-openjdk8.sh
dosbox-0.74.3-i586-2_SBo: DOS emulator/virtual machine for X11 and Unix  [ 2.6M]
Running dbgl-legacy.SlackBuild ...                                  ETA 00:10:??
env MAKEFLAGS='-j24 -l13' nice -n 5 bash ./dbgl-legacy.SlackBuild
Built ok:  dbgl-legacy-0.83-i586-7_SBo.txz                              00:10:12
Testing package dbgl-legacy-0.83-i586-7_SBo.txz ... done.
Test installing dbgl-legacy-0.83-i586-7_SBo.txz ...
dbgl-legacy-0.83-i586-7_SBo: a Java frontend for DOSBox ................ [ 6.9M]
Unmounting chroot ... done.
Backed up: dbgl-legacy-0.83-i586-6_SBo.txz
:-) desktop/dbgl-legacy Updated for git 3e9dbe9 "new maintainer; builds from source" (-:

================================================================================
SUMMARY                                                                 00:10:15
================================================================================

Logfile:     /var/log/repo/SBo/repo_2023-05-20_00:09:40.log
Runtime:     0h 0m 35s
Updated:     1
  desktop/dbgl-legacy/dbgl-legacy-0.83-i586-7_SBo.txz

Using tests from /usr/share/sbo-maintainer-tools/sbopkglint.d
Exploding /var/lib/repo/SBo/packages/15.0/i586/desktop/dbgl-legacy/dbgl-legacy-0.83-i586-7_SBo.txz to /tmp/sbopkglint.MPy7Qj ...OK
Running pre-doinst test...OK
Installing /var/lib/repo/SBo/packages/15.0/i586/desktop/dbgl-legacy/dbgl-legacy-0.83-i586-7_SBo.txz to /tmp/sbopkglint.MPy7Qj ...OK
Running test: 05-basic-sanity...OK
Running test: 10-docs...OK
Running test: 15-noarch...
OK
___ note: package might be a good candidate for noarch
Running test: 20-arch...OK
Running test: 25-lafiles...OK
Running test: 30-manpages...OK
Running test: 35-desktop...usr/share/applications/dbgl.desktop: warning: value "DOSBox Game Launcher" for key "Comment" in group "Desktop Entry" looks the same as that of key "GenericName"
OK
Running test: 40-newconfig...OK
Running test: 45-doinst...OK
Running test: 50-icons...OK
=== dbgl-legacy-0.83-i586-7_SBo.txz: All tests passed
```

```
# buildpkg x86_64 desktop/dbgl-legacy
sbolint: dbgl-legacy checks out OK
git repo: /var/lib/repo/SBo/slackbuilds
branch:   github
date:     Sat May 20 00:10:30 GMT 2023
revision: 8a5e5a82ae69e96399b0fda2813a0c8479dcbd44
title:    desktop/dbgl-legacy: new maintainer; builds from source

================================================================================
desktop/dbgl-legacy                                                     00:10:34
================================================================================

Calculating dependencies ... 
Hints for development/zulu-openjdk8:
  PRAGMA="x86arch=i686"
Dependency tree:
desktop/dbgl-legacy (update for git 8a5e5a8 "new maintainer; builds from source")
  system/dosbox (ok)
  development/zulu-openjdk8 (ok)
  development/apache-ant (ok)

----------------------------------------------------------------------- 00:10:35
Building desktop/dbgl-legacy (update for git 8a5e5a8 "new maintainer; builds from source")
Testing SlackBuild files ... done.
Downloading source files ...
Verifying source files ... done.
Installing dependencies ...
apache-ant-1.10.12-noarch-1_SBo: Java-based build tool ................. [  42M]
  Running profile script: /etc/profile.d/apache-ant.sh
zulu-openjdk8-8.0.362-x86_64-2_SBo: Open Implementation of JDK ......... [ 203M]
  Running profile script: /etc/profile.d/zulu-openjdk8.sh
dosbox-0.74.3-x86_64-2_SBo: DOS emulator/virtual machine for X11 and Uni [ 2.5M]
Running dbgl-legacy.SlackBuild ...                                  ETA 00:11:??
env MAKEFLAGS='-j24 -l13' nice -n 5 bash ./dbgl-legacy.SlackBuild
Built ok:  dbgl-legacy-0.83-x86_64-8_SBo.txz                            00:11:01
Testing package dbgl-legacy-0.83-x86_64-8_SBo.txz ... done.
Test installing dbgl-legacy-0.83-x86_64-8_SBo.txz ...
dbgl-legacy-0.83-x86_64-8_SBo: a Java frontend for DOSBox .............. [ 7.1M]
Unmounting chroot ... done.
Backed up: dbgl-legacy-0.83-x86_64-7_SBo.txz
:-) desktop/dbgl-legacy Updated for git 8a5e5a8 "new maintainer; builds from source" (-:

================================================================================
SUMMARY                                                                 00:11:03
================================================================================

Logfile:     /var/log/repo/SBo/repo_2023-05-20_00:10:34.log
Runtime:     0h 0m 29s
Updated:     1
  desktop/dbgl-legacy/dbgl-legacy-0.83-x86_64-8_SBo.txz

Using tests from /usr/share/sbo-maintainer-tools/sbopkglint.d
Exploding /var/lib/repo/SBo/packages/15.0/x86_64/desktop/dbgl-legacy/dbgl-legacy-0.83-x86_64-8_SBo.txz to /tmp/sbopkglint.c8MUen ...OK
Running pre-doinst test...OK
Installing /var/lib/repo/SBo/packages/15.0/x86_64/desktop/dbgl-legacy/dbgl-legacy-0.83-x86_64-8_SBo.txz to /tmp/sbopkglint.c8MUen ...OK
Running test: 05-basic-sanity...OK
Running test: 10-docs...OK
Running test: 15-noarch...
OK
___ note: package might be a good candidate for noarch
Running test: 20-arch...OK
Running test: 25-lafiles...OK
Running test: 30-manpages...OK
Running test: 35-desktop...usr/share/applications/dbgl.desktop: warning: value "DOSBox Game Launcher" for key "Comment" in group "Desktop Entry" looks the same as that of key "GenericName"
OK
Running test: 40-newconfig...OK
Running test: 45-doinst...OK
Running test: 50-icons...OK
=== dbgl-legacy-0.83-x86_64-8_SBo.txz: All tests passed
```

**Comment by aclemons** [comment]
LGTM

**Pull request merged**